train/compute/python/tools/validate_trace.py

```diff
@@ -58,6 +57,20 @@ def check_comms_node(n) -> bool:
                     has_pg_id = True
             return has_pg_id
 
+        def check_comms_node_new(n) -> bool:
```

Comment:
check_comms_node_old and  check_comms_node_new seems not very specific. We may introduce more schema change later. Can we use check_comms_node_less_than_1.1.0 and check_comms_node_1.1.0?